ArduCopter/Attitude.cpp

```diff
@@ -1,5 +1,21 @@
 #include "Copter.h"
 
+/*************************************************************
+ *  Attitude Rate controllers and timing
+ ****************************************************************/
+
+// update rate controllers and output to roll, pitch and yaw actuators
+//  called at 400hz by default
+void Copter::run_rate_controller()
+{
+    // set attitude and position controller loop time
+    attitude_control->set_dt(AP::scheduler().get_last_loop_time_s());
```

Comment:
It might be slightly better style to have a, "const float last_loop_time_s = AP::scheduler().get_last_loop_time_s()" and then pass this into the next two functions instead of calling the scheduler twice.  It should make no functional difference of course.

Also there's perhaps no need to use "AP::scheduler()." instead simply "scheduler." could be used.